src/pages/guides/eqp/v1/packages.md

```diff
@@ -0,0 +1,1056 @@
+|`admin_removed_from_store`|string|The EQP admin has removed the package from the store.|
+
+##### Technical Status
+
+|Value|Type|Description|
+|-----|----|-----------|
+|`draft`|string|The package has not been submitted for technical review.|
+|`in_automation`|string|The package has been submitted for technical review and is currently undergoing automated testing.|
+|`awaiting_manual_qa`|string|The package has passed all automated tests and is currently in the manual test queue.|
+|`in_manual_qa`|string|The package is currently undergoing manual testing.|
+|`approved`|string|The package has passed all tests.|
+|`rejected`|string|The package has failed automation or manual tests.|
+|`recalled`|string|The developer has recalled the package.|
+
+##### Marketing Status
+
+|Value|Type|Description|
+|-----|----|-----------|
+|`draft`|string|The package has not been submitted for marketing review.|
+|`awaiting_marketing_review`|string|The package has been submitted to the marketing review queue.|
+|`in_marketing_review`|string|The package is currently undergoing marketing review.|
+|`approved`|string|The package has passed the marketing review process.|
+|`rejected`|string|The package has failed the marketing review process.|
+|`recalled`|string|The developer has recalled the package.|
+
+## Package submissions
+
+```http
+POST /rest/v1/products/packages
+PUT /rest/v1/products/packages
+PUT /rest/v1/products/packages/:submission_id
+PUT /rest/v1/products/packages/:item_id
+```
+
+### Submit a package
+
+You can submit a package in either of the following ways:
+
+*  A single POST request with all required fields set. You must explicitly indicate that you are submitting for technical and marketing review using the `action` parameter.
+   *  If this package represents a newer version to an already existing extension, then also supply the `sku` parameter.
+*  A series of requests, typically in the following order:
+   1. A single POST request with the minimum required fields set and `action` set to `draft` in either `technical`, `marketing`, or both. This request accepts the new package and saves it on the Developer Portal for further updates. It returns a unique `submission_id` for subsequent PUT operations.
+   1. One or more PUT requests in which you configure the package parameters. In these requests, set `action` to `draft` in `technical`, `marketing`, or both.
+   1. A final PUT request indicating submission for `technical`, `marketing`, or both.
+
+You can update one or more parameters in `draft` mode.
+In this mode, the API checks only for basic type-validation issues.
+
+When the `action` field is set to `submit`, the API validates fields to ensure all
+required parameters are available on the Developer Portal to initiate the EQP process,
+and that parameters which depend upon each other match up correctly.
+
+<InlineAlert variant="info" slots="text"/>
+All `action` fields are optional. If not specified, `draft` is the default value.
```

Comment:
The backticks aren't rendering code format...

![Screen Shot 2022-04-01 at 2 15 23 PM](https://user-images.githubusercontent.com/13662379/161327437-31aa7fdd-0841-4462-bb94-2782210fcadd.png)